services/server/src/server/server.ts

```diff
@@ -324,74 +337,6 @@ export class Server {
   }
 }
 
```

Comment:
I imagine some people will assume running `dist/server.js`. For compatibility, should we add something like below? Would this cause any problems?

```ts
if (require.main === module) {
  import "./cli"
}
```

Reply:
I see we added so even if we run `node/dist/server.js` everything works as expected.

https://github.com/RaoulSchaffranek/sourcify/blob/4d77f987b0506d5cf946845e8fbf36cc395dd5ca/services/server/src/server/server.ts#L341-L343
```ts
if (require.main === module) {
  import("./cli");
}
```

It's ok, but I prefer to have only one entry point to reduce confusion.

E.g. in this Dockerfile running `"./dist/server/server.js"` works, but then from the `package.json` we use `node ./dist/server/cli.js`. I think it's confusing for new users
https://github.com/ethereum/sourcify/blob/cfc95f3473895c4541046743076e54cee50f226e/services/server/Dockerfile.debug#L11

Reply:
Hmm ok right. Let's just use `./cli.js`

Reply:
I don't have hard feelings for any option. Antoher option to keep backwards compatibility could be renaming `server.ts` to something like `app.ts` and then `cli.ts` to `server.ts`.

One additional comment about `require.main === module`: This [StackOverflow answer](https://stackoverflow.com/questions/45136831/node-js-require-main-module#answer-60309682) mentions some compatibility issue when the module is loaded as a EcmaScript module. Not sure, if this is relevant and how this would affect the execution.